core/iwasm/include/wasm_c_api.h

```diff
@@ -527,6 +527,7 @@ typedef struct WASMModuleCommon *wasm_module_t;
 #define LOAD_ARGS_OPTION_DEFINED
 typedef struct LoadArgs {
     char *name;
+    bool no_wasm_binary_copy;
```

Comment:
Or is it better to rename it to `clone_wasm_binary`? If yes, we had better set it true in wasm_module_new to keep compatibility:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/common/wasm_c_api.c#L2344

And we can also keep the compatibility for wasm_runtime_load_ex when we enable `LoadArgs->clone_wasm_binary = true` in the future, since the current behavior is like that `LoadArgs->clone_wasm_binary` is set as false now.